src/keywords/rFactorRangesVector.cpp

```diff
@@ -0,0 +1,109 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// Copyright (c) 2023 Team Dissolve and contributors
+
+#include "keywords/rFactorRangesVector.h"
+#include "base/lineParser.h"
+#include "classes/coreData.h"
+
+RFactorRangesVector::RFactorRangesVector(std::vector<Range> &data) : KeywordBase(typeid(this)), data_(data) {}
```

Comment:
In line with earlier suggestions, and adding on the missing `Keyword` suffix to denote it as such, the class should be called `RangeVectorKeyword`:
```suggestion
RangeVectorKeyword::RangeVectorKeyword(std::vector<Range> &data) : KeywordBase(typeid(this)), data_(data) {}
```